src/testrunner/Microsoft.NET.Test.Runner/Microsoft.NET.Test.Runner.sln

```diff
@@ -0,0 +1,32 @@
+﻿
```

Comment:
@ViktorHofer I need to build the test runner package during the build(using private dotnet), can you point me to a good place?